src/components/RoutesRow/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Box, IconButton, Typography } from '@mui/material';
+import DriverAvatar from '../DriverAvatar';
+import EditIcon from '@mui/icons-material/Edit';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import { COLORS } from '@/constants/colors';
+import { FONT } from '@/constants/font';
+import Status from '../Status';
+
+interface RoutesRowProps {
+  routeId: string;
+  date: string;
+  driverFirstName: string;
+  driverLastName: string;
+  driverPhone: string;
+  stopsCount: number;
+  workingHours: string;
+  distance: number;
+  status: 'completed' | 'failed' | 'not_arrived' | 'at_risk' | 'upcoming';
```

Comment:
is something repeat its a red flag that it should not be in the component, move to to constant